packages/renderer/src/lib/pod/PodsList.svelte

```diff
@@ -241,13 +240,12 @@ function errorCallback(pod: PodInfoUI, errorMessage: string): void {
       <thead>
         <tr class="h-7 uppercase text-xs text-gray-600">
           <th class="whitespace-nowrap w-5"></th>
-          <th class="px-2 w-5"
-            ><input
-              type="checkbox"
+          <th class="px-2 w-5">
+            <Checkbox
+              bind:checked="{selectedAllCheckboxes}"
               indeterminate="{selectedItemsNumber > 0 && !selectedAllCheckboxes}"
-              bind:checked="{allChecked}"
-              on:click="{event => toggleAllPods(event.currentTarget.checked)}"
-              class="cursor-pointer invert hue-rotate-[218deg] brightness-75" /></th>
+              on:click="{event => toggleAllPods(event.currentTarget)}" />
```

Comment:
I'm not sure to understand how the on:click on <Checkbox is providing the event that is from the input being internal where we provide an empty on:click

```
<label>
  <input
    type="checkbox"
...
    on:click />
```

Reply:
probably should do:
```
on:click="{event => toggleCheckboxContainerGroup(event, containerGroup)}" />
```

and
```
function toggleAllPods(event: MouseEvent & {target: HTMLInputElement}) {
  const checked =element.target.checked;
```